tests/Unit/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/TestFunctions.py

```diff
@@ -0,0 +1,172 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+# Functions for testing FishboneMoncriefDisk.cpp
+def delta(r_sqrd, m, a):
+    return r_sqrd - 2.0 * m * np.sqrt(r_sqrd) + a**2
+
+
+def sigma(r_sqrd, sin_theta_sqrd, a):
+    return r_sqrd + (1.0 - sin_theta_sqrd) * a**2
+
+
+def ucase_a(r_sqrd, sin_theta_sqrd, m, a):
+    return ((r_sqrd + a**2)**2
+            - delta(r_sqrd, m, a) * sin_theta_sqrd * a**2)
+
+
+def boyer_lindquist_gtf(r_sqrd, sin_theta_sqrd, m, a):
+    return (-2.0 * m * np.sqrt(r_sqrd) * a * sin_theta_sqrd /
+            sigma(r_sqrd, sin_theta_sqrd, a))
+
+
+def boyer_lindquist_gff(r_sqrd, sin_theta_sqrd, m, a):
+    return (ucase_a(r_sqrd, sin_theta_sqrd, m, a) * sin_theta_sqrd /
+            sigma(r_sqrd, sin_theta_sqrd, a))
+
+
+def boyer_lindquist_r_sqrd(x, a):
+    half_diff = 0.5 * (x[0]**2 + x[1]**2 + x[2]**2 - a**2)
+    return half_diff + np.sqrt(half_diff**2 + a**2 * x[2]**2)
+
+
+def boyer_lindquist_sin_theta_sqrd(z_sqrd, r_sqrd):
+    return 1.0 - z_sqrd / r_sqrd
+
+
+def kerr_schild_h(x, m, a):
+    r_sqrd = boyer_lindquist_r_sqrd(x, a)
+    return m * r_sqrd * np.sqrt(r_sqrd) / (r_sqrd**2 + a**2 * x[2]**2)
+
+
+def kerr_schild_spatial_null_form(x, m, a):
+    r_sqrd = boyer_lindquist_r_sqrd(x, a)
+    r = np.sqrt(r_sqrd)
+    denom = 1.0 / (r_sqrd + a**2)
+    return np.array([(r * x[0] + a * x[1]) * denom,
+                     (r * x[1] - a * x[0]) * denom,
+                     x[2] / r])
+
+
+def kerr_schild_lapse(x, m, a):
+    null_vector_0 = -1.0
+    return np.sqrt(1.0 / (1.0 + 2.0 * kerr_schild_h(x, m, a) *
+                          null_vector_0 * null_vector_0))
+
+
+def kerr_schild_shift(x, m, a):
+    null_vector_0 = -1.0
+    prefactor = (-2.0 * kerr_schild_h(x, m, a) * null_vector_0 *
+                 kerr_schild_lapse(x, m, a)**2)
+    null_form = kerr_schild_spatial_null_form(x, m, a)
+    return np.array([prefactor * null_form[0],
```

Comment:
`prefactor * null_form` will do the same thing right? Since `kerr_child_spatial_null_form` is already a `np.array`